content/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -204,35 +204,37 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
         </GalleryContainer>
       )}
       {(!isFullSupportBrowser || isSmallGallery) && (
-        <ImageCardList data-test-id="image-search-results-container">
-          {imagesWithDimensions.map((result: GalleryImageProps) => (
-            <li key={result.id}>
-              <Space
-                $h={{ size: 'l', properties: ['margin-right'] }}
-                $v={{ size: 'l', properties: ['margin-bottom'] }}
-              >
-                <ImageCard
-                  id={result.id}
-                  workId={result.source.id}
-                  image={{
-                    contentUrl: result.src,
-                    width: result.width * 1.57,
-                    height: result.height * 1.57,
-                    alt: result.source.title,
-                  }}
-                  onClick={event => {
-                    if (isSmallGallery) {
-                      event.preventDefault();
-                      setExpandedImage(result);
-                      setIsActive(true);
-                    }
-                  }}
-                  layout="fixed"
-                />
-              </Space>
-            </li>
-          ))}
-        </ImageCardList>
+        <div data-testid="image-search-results-container">
+          <ImageCardList>
```

Comment:
Let's review later if this change is still needed.

Reply:
wait, just to confirm because I had to remind myself what this was, this change was about adding a `div` element to wrap `ImageCardList` (`ul`) in as one of the selectors was looking for `page.getByTestId('image-search-results-container') > ul > ...` and in this case, the test-id used to be on the `ul` directly and so it failed.

Reply:
I only remember changing l.194 from test-id to testiid so that playwright can findByTestid

Reply:
I retested it and it's still needed so we can leave as is! More aligned that way anyway.